public/javascripts/SVValidate/src/keyboard/Keyboard.js

```diff
@@ -192,6 +324,39 @@ function Keyboard(menuUI) {
                     menuUI.unsureButton.removeClass("validate");
                     status.keyPressed = false;
                     break;
+                // "1" key
+                case 49:
```

Comment:
I don't think that we need these in both key up and key down?

Reply:
@misaugstad I was thinking we need "1" key in both as in documentkeyDown when they press on the key and documentKeyUp so that it resets the value of severity after submitting that validation - not sure if this is actually how it works, I have not figured out how to reset the severity after submitting

Reply:
done